app/utils/sentry.ts

```diff
@@ -147,16 +146,16 @@ function capture(captureFunc: () => void, config?: ClientConfig) {
         }
 
         if (hasUserContext) {
-            console.warn('Capturing with Sentry at ' + getDsn() + '...'); // eslint-disable-line no-console
+            logWarning('Capturing with Sentry at ' + getDsn() + '...');
 
             captureFunc();
         } else {
-            console.warn('No user context, skipping capture'); // eslint-disable-line no-console
+            logWarning('No user context, skipping capture');
         }
     } catch (e) {
         // Don't want this to get into an infinite loop again...
-        console.warn('Exception occured while sending to Sentry'); // eslint-disable-line no-console
-        console.warn(e); // eslint-disable-line no-console
+        logWarning('Exception occured while sending to Sentry');
+        logWarning(e);
```

Comment:
should we log this as error instead ?

Reply:
I wanted to keep the "original behaviour" without many changes, but seems clear to make this change.